manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -401,15 +400,13 @@ spec:
               mountPath: /csi
             - name: registration-dir
               mountPath: /registration
-          ports:
-            - containerPort: 9809
-              name: healthz
           livenessProbe:
-            httpGet:
-              path: /healthz
-              port: healthz
-            initialDelaySeconds: 5
-            timeoutSeconds: 5
+            exec:
+              command:
+              - /csi-node-driver-registrar
+              - --kubelet-registration-path=/var/lib/kubelet/plugins/csi.vsphere.vmware.com/csi.sock
+              - --mode=kubelet-registration-probe
+            initialDelaySeconds: 3
```

Comment:
Why are we defaulting `timeoutSeconds` to 1 second after this change ? Just wanted to know the reasoning

Reply:
I just used the example provided here - https://github.com/kubernetes-csi/node-driver-registrar#health-check-with-an-exec-probe